### PR TITLE
Madenearme fixes

### PR DESCRIPTION
**Pull request by hangy**
Based on #2690, improves the HTML of the made near me sites a bit.

### CODE REVIEW

File: madenearme/madenearme-uk.html [hunk 1]
```diff
@@ -115,12 +97,11 @@ <h2>Contact</h2>
 
 <p>
 &rarr; <a href="https://twitter.com/openfoodfactsuk">Twitter</a><br/>
-&rarr; <a href="https://plus.google.com/u/0/+OpenfoodfactsOrg">Google+</a><br />
 &rarr; <a href="https://www.facebook.com/OpenFoodFacts">Facebook</a> + <a href="https://www.facebook.com/groups/374350705955208/">Open Food Facts contributors group</a><br />
 </p>
 
 
-<p>You can add products easily with our <a href="<a href="https://itunes.apple.com/app/open-food-facts/id588797948">iPhone</a>,
+<p>You can add products easily with our <a href="https://itunes.apple.com/app/open-food-facts/id588797948">iPhone</a>,
```
Comment (teolemon) [inline madenearme/madenearme-uk.html]:
```suggestion
<p>You can add products easily with our <a href="https://apps.apple.com/app/open-food-facts/id588797948">iPhone</a>,
```

File: madenearme/madenearme-fr.html [hunk 2]
```diff
@@ -122,7 +92,6 @@ <h2>On en discute ?</h2>
 
 <p>
 &rarr; <a href="https://twitter.com/openfoodfactsfr">Twitter</a><br/>
```
Comment (teolemon) [inline madenearme/madenearme-fr.html]:
```suggestion
&rarr; <a href="https://twitter.com/openfoodfactsfr">Twitter</a><br/>
&rarr; <a href="https://www.instagram.com/open.food.facts/">Instagram</a><br />
```

File: madenearme/madenearme-uk.html [hunk 3]
```diff
@@ -115,12 +97,11 @@ <h2>Contact</h2>
 
 <p>
 &rarr; <a href="https://twitter.com/openfoodfactsuk">Twitter</a><br/>
```
Comment (teolemon) [inline madenearme/madenearme-uk.html]:
```suggestion
&rarr; <a href="https://twitter.com/openfoodfactsuk">Twitter</a><br/>
&rarr; <a href="https://www.instagram.com/open.food.facts/">Instagram</a><br />
```

File: scripts/generate_madenearme_page.pl [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+if ((not defined $cc) or (not defined $lc)) {
+	die("Pass country code (or world) and language code as arguments.\n");
+}
+else {
+		if (defined $country_codes{$cc}) {
+			$country = $country_codes{$cc};
+		}
+		else {
+			$country = "en:world";
```
Comment (VaiTon) [inline scripts/generate_madenearme_page.pl]: I think we should throw an error here and not just pretend that world is selected

File: scripts/generate_madenearme_page.pl [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+# We want products with emb codes
+$query_ref->{"emb_codes_tags"} = { '$exists' => 1 };
+
+$request_ref->{map_options} = $map_options{$cc} || "";
+
+my $map_html = search_and_map_products($request_ref, $query_ref, $graph_ref);
+
+
+$html =~ s/<HEADER>/$header/;
```
Comment (VaiTon) [inline scripts/generate_madenearme_page.pl]: Could we use Template::HTML here?

File: scripts/generate_madenearme_page.pl (hunk 4 above)
Comment (hangy) [reply scripts/generate_madenearme_page.pl]: This line belonged to the other PR

File: scripts/generate_madenearme_page.pl (hunk 5 above)
Comment (hangy) [reply scripts/generate_madenearme_page.pl]: This line belonged to the other PR

**Pull request merged**